packages/react-next/src/components/Checkbox/examples/Checkbox.Basic.Example.tsx

```diff
@@ -1,22 +1,25 @@
 import * as React from 'react';
 import { Checkbox } from '@fluentui/react-next/lib/Checkbox';
 import { Stack } from '@fluentui/react-next/lib/Stack';
+import { ThemeProvider } from '@fluentui/react-next/lib/Styling';
 
 // Used to add spacing between example checkboxes
 const stackTokens = { childrenGap: 10 };
 
 export const CheckboxBasicExample: React.FunctionComponent = () => {
   // These checkboxes are uncontrolled because they don't set the `checked` prop.
   return (
-    <Stack tokens={stackTokens}>
-      <Checkbox label="Unchecked checkbox (uncontrolled)" onChange={_onChange} />
+    <ThemeProvider>
```

Comment:
I think it's fine for this PR but we should probably find a way to inject the `ThemeProvider` at the example layer and not in each example.

Reply:
Agree that it's fine to get this going here as a start, but we'll likely want to revisit before ship.

Reply:
maybe add a TODO (xugao)

Reply:
fixed, now it's automatically wrapped on all example stories